### ISSUE TITLE
Row evolution with goal crash

### ISSUE THREAD

**Issue by sb4m** [matomo-org/matomo#11526]
Hello,

I have an issue when I want to open Row evolution on this screen

![superimmoneuf_-_thursday__march_23__2017_-_web_analytics_reports_-_piwik](https://cloud.githubusercontent.com/assets/66470/24298144/68d92446-10a5-11e7-96ed-5915a0946def.png)

I get

![superimmoneuf_-_jeudi_23_mars_2017_-_rapports_des_statistiques_web__-_piwik](https://cloud.githubusercontent.com/assets/66470/24298079/34336a44-10a5-11e7-9f98-1a55df771395.png)

The ajax call is  : http://piwik.XXXX.com/index.php?date=2017-03-23&apiMethod=Referrers.getReferrerType&label=%40Campagnes&disableLink=1&module=CoreHome&action=getRowEvolutionPopover&colors=%7B%22backgroundColor%22%3A%22%23ffffff%22%2C%22lineColor%22%3A%22%23162c4a%22%2C%22minPointColor%22%3A%22%23ff7f7f%22%2C%22maxPointColor%22%3A%22%2375bf7c%22%2C%22lastPointColor%22%3A%22%2355aaff%22%7D&idGoal=3&idSite=12&period=day

I get an error page

If I remove &idGoal=3 all works

How can I get more info to debug this ?

**Comment by sb4m** [comment]
I had more info in my log

```
DEBUG: /home/piwik/piwik/plugins/API/RowEvolution.php(326): Requested report Referrers.getReferrerType for Website id=12 not found in the list of available reports. #0 /home/piwik/piwik/plugins/API/RowEvolution.php(54): Piwik\Plugins\API\RowEvolution-&gt;getRowEvolutionMetaData('12', 'day', '2017-02-26,2017...', 'Referrers', 'getReferrerType', false, '3', false) #1 /home/piwik/piwik/plugins/API/API.php(438): Piwik\Plugins\API\RowEvolution-&gt;getRowEvolution('12', 'day', '2017-02-26,2017...', 'Referrers', 'getReferrerType', '@Moteurs%20de%2...', false, false, false, '3', true, true, false) #2 [internal function]: Piwik\Plugins\API\API-&gt;getRowEvolution('12', 'day', '2017-02-26,2017...', 'Referrers', 'getReferrerType', '@Moteurs%20de%2...', false, false, false, '3', true, true, false) #3 /home/piwik/piwik/core/API/Proxy.php(219): call_user_func_array(Array, Array) #4 /home/piwik/piwik/core/API/Request.php(236): Piwik\API\Proxy-&gt;call('\\Piwik\\Plugins\\...', 'getRowEvolution', Array) #5 /home/piwik/piwik/plugins/CoreHome/DataTableRowAction/RowEvolution.php(171): Piwik\API\Request-&gt;process() #6 /home/piwik/piwik/plugins/CoreHome/DataTableRowAction/RowEvolution.php(111): Piwik\Plugins\CoreHome\DataTableRowAction\RowEvolution-&gt;loadEvolutionReport() #7 /home/piwik/piwik/plugins/CoreHome/Controller.php(250): Piwik\Plugins\CoreHome\DataTableRowAction\RowEvolution-&gt;__construct(12, Object(Piwik\Date), NULL) #8 /home/piwik/piwik/plugins/CoreHome/Controller.php(216): Piwik\Plugins\CoreHome\Controller-&gt;makeRowEvolution(false) #9 [internal function]: Piwik\Plugins\CoreHome\Controller-&gt;getRowEvolutionPopover() #10 /home/piwik/piwik/core/FrontController.php(542): call_user_func_array(Array, Array) #11 /home/piwik/piwik/core/FrontController.php(137): Piwik\FrontController-&gt;doDispatch(NULL, NULL, NULL) #12 /home/piwik/piwik/core/dispatch.php(34): Piwik\FrontController-&gt;dispatch() #13 /home/piwik/piwik/index.php(27): require_once('/home/piwik/piw...') #14 {main}

How can i reprocess this missing report .

**Comment by mattab** [comment]
Hi there, please upgrade to Piwik 3.0.3 where this should work. If you still have the problem please re-open this issue.

**Comment by sb4m** [comment]
Hello,
I still have the issue. 
How can I fix this pb, help in anyway ?

**Comment by mattab** [comment]
Fixed #12378 - it will be released in 3.5.0-b2 in a few hours
